includes/abstracts/abstract.llms.post.model.php

```diff
@@ -958,10 +958,17 @@ public function set_bulk( $model_array, $wp_error = false ) {
 
 		foreach ( $model_array as $key => $val ) {
 
+			$is_post_property = in_array( $key, $post_properties );
+
+			// sanitize the post properties keys by removing the "post_" prefix.
+			if ( $is_post_property && "post_" === substr( $key, 0, 5 ) ) {
```

Comment:
single quotes instead of double (check coding standards via `composer run-script check-cs`

Fix and resubmit please

Reply:
Yes sorry, I ran the check on the lifterlms-rest dir.

Reply:
Fixed in 0a2b7d525cf0e4ce67a1fc74a6a8013309c519fb